creator-node/test/audiusUsers.js

```diff
@@ -66,3 +84,88 @@ describe('test AudiusUsers', function () {
       .expect(200)
   })
 })
+
+describe('tests /audius_users/metadata metadata upload with actual ipfsClient', function () {
```

Comment:
why is this a new describe block?

Reply:
the above tests run with the ipfsMock, and in this describe block, it runs the it() tests with the actual ipfs client. this way, we can inherently test the ipfs client and mimic the actual environment

Reply:
could you put a comment above this describe block explicitly stating that? just so it's obvious to someone else reading it for the first time